src/views/Project/index.jsx

```diff
@@ -111,24 +118,22 @@ export default class Project extends Component {
       [];
 
     return (
-      <Fragment>
-        <header>
-          <Typography variant="display2" align="center">
-            {project.name}
+      <div className={classes.root}>
+        <Typography variant="display2" align="center" color="primary">
+          {project.name}
+        </Typography>
+        <Typography variant="subheading" align="center" color="textSecondary">
+          Bugs & Issues
+        </Typography>
```

Comment:
Is it possible to have an Appbar or something to make the toolbar consistent with the projects view? I'm thinking of something similar to the home page but without the search bar. The current look feels rather plain and not consistent with the other page.

Aside, I think it would be cool to have either a back button or a Codetribute button that would take us back to the home page. For example, if you navigate to https://tools.taskcluster.net/groups/, clicking on "TASKCLUSTER" (top left) will navigate you to the home page. This could be implemented later.